packages/discovery-react-components/src/components/SearchFacets/utils/searchFacetInterfaces.ts

```diff
@@ -32,3 +38,9 @@ export interface CollectionItem {
   id: string;
   label: string;
 }
+
+export interface QueryAggregationWithName extends DiscoveryV2.QueryAggregation {
+  field?: string;
+  count?: number;
+  name?: string;
```

Comment:
Doesn't he SDK already define `name`? If not, we should be pushing that into the SDK

Reply:
The SDK defines `name` in `ComponentSettingsAggregation` but not for the query aggregation. Yes, I wanted to get something working here at least, but I agree it should be defined in the SDK. Should I just open a PR here (https://github.com/watson-developer-cloud/node-sdk) to add it in?

Reply:
We should probably push it to the API definitions repo so it gets added to all languages